proarc-common/src/test/java/cz/cas/lib/proarc/common/dao/empiredb/EmpireWorkflowMaterialDaoTest.java

```diff
@@ -286,6 +289,15 @@ public void testView() throws Exception {
         assertNull(ms.get(0).getTaskId());
         assertEquals(MaterialType.FOLDER, ms.get(0).getType());
         assertNull(ms.get(0).getWay());
+
+        // view job's material with specific type
+        for (MaterialType type : MaterialType.values()) {
+            filter = new MaterialFilter();
+            filter.setJobId(BigDecimal.ONE);
+            filter.setType(type);
+            assertTrue(dao.view(filter).stream().allMatch(material-> type.equals(material.getType())));
```

Comment:
Přidal bych do assertTrue informaci, který materiál neodpovídá očekáváním. Z test reportu to teď nejde poznat.